.github/workflows/build.yml

```
@@ -236,8 +236,17 @@ jobs:
   publish:
     name: Publish to PyPi
     if: github.repository == 'Eeems-Org/remarkable-update-fuse' && github.event_name == 'release' && startsWith(github.ref, 'refs/tags')
-    needs: [build-sdist]
+    needs:
+     - build-sdist
```

yamllint (1.35.1)

[error] wrong indentation: expected 6 but found 5

(indentation)